hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -363,6 +365,199 @@ public OMResponse handle(OMRequest request) {
+        break;
+      default:
+        // As all request types are not changed so we need to call handle
+        // here.
+        return handle(omRequest);
+      }
+      responseBuilder.setSuccess(true);
+    } catch (IOException ex) {
+      responseBuilder.setSuccess(false);
+      responseBuilder.setStatus(exceptionToResponseStatus(ex));
+      if (ex.getMessage() != null) {
+        responseBuilder.setMessage(ex.getMessage());
+      }
+    }
+    return responseBuilder.build();
+  }
+
+
+  private OMRequest handleCreateVolumeStart(OMRequest omRequest)
+      throws IOException {
+    try {
+      OzoneManagerProtocolProtos.VolumeInfo volumeInfo =
+          omRequest.getCreateVolumeRequest().getVolumeInfo();
+      OzoneManagerProtocolProtos.VolumeList volumeList =
+          impl.startCreateVolume(OmVolumeArgs.getFromProtobuf(volumeInfo));
+
+      CreateVolumeRequest createVolumeRequest =
+          CreateVolumeRequest.newBuilder().setVolumeInfo(volumeInfo)
+              .setVolumeList(volumeList).build();
+      return omRequest.toBuilder().setCreateVolumeRequest(createVolumeRequest)
+          .build();
+    } catch (IOException ex) {
+      throw ex;
+    }
+  }
+
+  private CreateVolumeResponse handleCreateVolumeApply(OMRequest omRequest)
+      throws IOException {
+    try {
+      OzoneManagerProtocolProtos.VolumeInfo volumeInfo =
+          omRequest.getCreateVolumeRequest().getVolumeInfo();
+      OzoneManagerProtocolProtos.VolumeList volumeList =
+          omRequest.getCreateVolumeRequest().getVolumeList();
+      impl.applyCreateVolume(OmVolumeArgs.getFromProtobuf(volumeInfo),
+          volumeList);
+    } catch (IOException ex) {
+      throw ex;
+    }
+    return CreateVolumeResponse.newBuilder().build();
+  }
+
+  private OMRequest handleSetVolumePropertyStart(OMRequest omRequest)
+      throws IOException {
+    SetVolumePropertyRequest setVolumePropertyRequest =
+        omRequest.getSetVolumePropertyRequest();
+    String volume = setVolumePropertyRequest.getVolumeName();
+    OMRequest newOmRequest = null;
+    if (setVolumePropertyRequest.hasQuotaInBytes()) {
+      long quota = setVolumePropertyRequest.getQuotaInBytes();
+      OmVolumeArgs omVolumeArgs = impl.startSetQuota(volume, quota);
+      SetVolumePropertyRequest newSetVolumePropertyRequest =
+          SetVolumePropertyRequest.newBuilder().setVolumeName(volume)
+              .setVolumeInfo(omVolumeArgs.getProtobuf()).build();
+      newOmRequest =
+          omRequest.toBuilder().setSetVolumePropertyRequest(
+              newSetVolumePropertyRequest).build();
+    } else {
+      String owner = setVolumePropertyRequest.getOwnerName();
+      OmVolumeOwnerChangeResponse omVolumeOwnerChangeResponse =
+          impl.startSetOwner(volume, owner);
+      // If volumeLists become large and when writing to disk we might take
+      // more space if the lists become very big in size. We might need to
+      // revisit this if it becomes problem
```

Comment:
Is this list stored in a separate table? I think comment text needs fixing, looks off.

Reply:
Yes. userTable.